spec/amber/cli/commands/init_spec.cr

```diff
@@ -1,88 +1,130 @@
 require "../../../spec_helper"
+require "../../../support/helpers/cli_helper"
+
+include CLIHelper
 
 module Amber::CLI
-  begin
-    describe MainCommand::New do
-      context "application structure" do
-        it "creates amber directory structure" do
-          MainCommand.run ["new", TESTING_APP]
-
-          Dir.exists?(TESTING_APP).should be_true
-          Amber::CLI::Spec.dirs(TESTING_APP).sort.should eq Amber::CLI::Spec.dirs(APP_TPL_PATH).sort
-          Amber::CLI::Spec.db_yml["pg"].should_not be_nil
-          shard_yaml = Amber::CLI::Spec.shard_yml
-          shard_yaml["dependencies"]["pg"].should_not be_nil
-          shard_yaml["dependencies"]["amber"]["version"].should eq Amber::VERSION
-          Amber::CLI::Spec.amber_yml["language"].should eq "slang"
-          File.read("#{TESTING_APP}/.amber_secret_key").size.should eq 44
-          Amber::CLI::Spec.cleanup
+  describe MainCommand::New do
```

Comment:
Same thing here, `describe "amber new #{TESTING_APP}" do` and remove the context below.